common/src/test/java/org/opensearch/sdk/SdkClientTests.java

```diff
@@ -86,162 +87,163 @@ public CompletionStage<SearchDataObjectResponse> searchDataObjectAsync(SearchDat
                 return CompletableFuture.completedFuture(searchResponse);
             }
         });
+        sdkClient = new SdkClient(sdkClientImpl);
```

Comment:
I assume we can use `sdkClient` like this way in algorithms module as we can't import SDKClientFactory class to algorithm module?

Reply:
Yes but you'd still need the LocalClusterIndicesClient.  One possibility is moving that default client to the `common` module since it doesn't need the Jackson stuff that the DDB Client does.

Reply:
That could be a easy workaround. Do you want to address that in this PR? Or in another PR?

Reply:
Probably another PR. I may make another attempt to solve the jackson problems.